docusaurus/sidebars.js

```diff
@@ -742,25 +765,22 @@ const sidebars = {
             updated: true,
           },
         },
-        'cloud/projects/collaboration',
-        'cloud/projects/runtime-logs',
+        "cloud/projects/collaboration",
+        "cloud/projects/runtime-logs",
       ],
     },
     {
       type: "category",
       collapsed: false,
       label: "Deployments",
-      items: [
-        "cloud/projects/deploys",
-        "cloud/projects/deploys-history",
-      ],
+      items: ["cloud/projects/deploys", "cloud/projects/deploys-history"],
```

Comment:
This is an improvement suggested by Prettier, and while it's a good idea for JavaScript code, in our case it reduces readability IMHO. Could you please change it back to 4 lines like in the original content?